calibration/CTDGVM/CGINS-CTDGVM-09269__20190408.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9269,CC_calibration_date,1554681600000,sensor calibration date (milliseconds since 1970-01-01)

```

Comment:
Date looks good and instrument exists in bulk_load

Reply:
good